app/genesis.go

```diff
@@ -67,25 +69,26 @@ func NewDefaultGenesisState() GenesisState {
 	authGenesis.Params.TxSizeCostPerByte = 5
 	stakingGenesis.Params.BondDenom = denom
 	stakingGenesis.Params.HistoricalEntries = 1000
-	mintGenesis.Params.BlocksPerYear = 10519200 // target 3-second block time
+	mintGenesis.Params.BlocksPerYear = 31557600 // target 1-second block time
 	mintGenesis.Params.MintDenom = denom
 	govGenesis.Params.MinDeposit = sdk.NewCoins(
 		sdk.NewCoin(denom, sdk.TokensFromConsensusPower(1000, sdk.DefaultPowerReduction)),
 	)
```

Comment:
<img width="847" alt="image" src="https://github.com/user-attachments/assets/b61e7139-a0cd-4e22-adc2-45b7d3fb42e2">

Need to adjust ExpeditedMinDeposit as well since there is a logic to validate it. Default on gov module it uses 5x from MinDeposit.